### PR TITLE
Create bugreport_23012023.md

### PR DESCRIPTION
**Pull request by SergeiRaik**

### CODE REVIEW

**Review by Zhenya-Barannik — COMMENTED** [review]
Hey. Please elaborate more on some items:
1, 2 — In what cases do you want to move the notebook and what do you try to achieve this way? I am not sure what you mean by creating the env, your environment should not be related to this issue generally.  
3  — I will check if it is possible to handle a coma.
4 — Also please elaborate more on what you would like to see in what cases. 
5 — Which name do you believe to be correct?
6 — Mind providing a link to the function you want to be implemented?

File: bugreport_23012023.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+1. After creating the env, notebook can't be moved to another folder
+2. When you create the env in the source folder, you have to change the working directory before you run the jupyter lab
+3. Decimal comma is not recognized
+4. LGC proficiency testing reports require alternative number of decimal points. One have to look for those in .json file
+5. Not sure if Alzymologist Oy Lab is a legal name.
```
Comment (Slesarew) [inline bugreport_23012023.md]: `Alzymologist Oy` is legal company name. I suppose appending the word "laboratory" in some form is legit language thing?

**Comment by Slesarew** [comment]
Also github has "issues" tab for these bug reports and feature requests on top, although this is fine too I suppose.

**Comment by SergeiRaik** [comment]
> Also github has "issues" tab for these bug reports and feature requests on top, although this is fine too I suppose.

Good point, we'll do that next time.

File: bugreport_23012023.md (hunk 1 above)
Comment (SergeiRaik) [reply bugreport_23012023.md]: Ok, I guess so.

**Comment by SergeiRaik** [comment]
> Hey. Please elaborate more on some items:
> 1, 2 — In what cases do you want to move the notebook and what do you try to achieve this way? I am not sure what you mean by creating the env, your environment should not be related to this issue generally.
> 3 — I will check if it is possible to handle a coma.
> 4 — Also please elaborate more on what you would like to see in what cases.
> 5 — Which name do you believe to be correct?
> 6 — Mind providing a link to the function you want to be implemented?

1,2 - It might be just a matter of habits. The case was that when I installed the notebook, I manually moved it to the root folder and had problems with an updated version. Being a less experienced command-line user, I'm more used to dragging and dropping things. 
Again, If we don't plan to elaborate this project into something bigger, that's fine as it is.
4. We might leave it in its current state. Just noted that in some cases values are presented differently, so proper handling of .json files is needed.
6. EBC 9.6 & 8.5

**Comment by Zhenya-Barannik** [comment]
> 1,2 - It might be just a matter of habits. The case was that when I installed the notebook, I manually moved it to the root folder and had problems with an updated version. Being a less experienced command-line user, I'm more used to dragging and dropping things. Again, If we don't plan to elaborate this project into something bigger, that's fine as it is. 4. We might leave it in its current state. Just noted that in some cases values are presented differently, so proper handling of .json files is needed. 6. EBC 9.6 & 8.5

The code in the notebook expects the `source/json/reports` folders to be in the same directory. So you should not move the notebook or those folders independently from each other. If you want to use two notebooks in parallel you can copy the notebook and use it inside the same folder. The only possible problem that I can think of is interference between default JSON files. You can solve this possible issue easily,  just change `TEMPORARY_NAME` constant:

 Open a copy of the notebook (for example `Distillation2.ipynb` )  and change `TEMPORARY_NAME`. For example, make it
 ```TEMPORARY_NAME = "temp2"```

**Pull request merged**